outline.md

```diff
@@ -0,0 +1,17 @@
+## Motivation
+
+Introduce the document by describing the motivations for producing a consolidated set of software citation principles. This section is basically an overview of the different use cases/scenarios where it would be desirable to have a way to cite software.
+
+## Related work
+
+In this section we discuss related work in the community. This is basically an overview of the documentation phased we went through as a group and will be heavily based on the collection of summaries we have of existing practices.
+
```

Comment:
I agree with npch that either here, or at the beginning of the document, there should be some contextualization. A short explanation of the genre of the document (a la: Six Provocations for Big Data, Joint Declaration of Data Citation Principles, Plos' Ten Simple Rules), explain how related to Force11 and Data Citation Principles specifically, timeline of development, and perhaps a generic statement on the breadth of expertise of the authors to give credence to the document. It appears that this the same idea as the "Preamble" section in the Data Citation Principles.